broker/lib/fabrik/DirectorManager.js

```diff
@@ -668,7 +691,7 @@ class DirectorManager extends BaseManager {
     const backupStartedAt = new Date().toISOString();
     const data = _
       .chain(opts)
-      .pick('service_id', 'plan_id', 'organization_guid', 'instance_guid', 'username')
+      .pick('deployment', 'service_id', 'plan_id', 'organization_guid', 'instance_guid', 'username')
```

Comment:
is the  `data.deployment` used ? 
Its being set again at https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/275/files#diff-b2c19de3bcc8e2e1fb90955f336cdbaeR757
and is not used anywhere else above.
If so can we remove it from here ?

Reply:
Okay. would check if data.deployment is nowhere used. If not would remove.